dataactbroker/handlers/fileHandler.py

```diff
@@ -27,7 +27,7 @@
 from dataactcore.interfaces.db import GlobalDB
 from dataactcore.interfaces.function_bag import (
     create_jobs, get_error_metrics_by_job_id, get_fabs_meta, mark_job_status, get_last_validated_date,
-    get_lastest_certified_date, get_certification_deadline, get_time_period)
+    get_lastest_published_date, get_certification_deadline, get_time_period)
```

Comment:
So now that you've renamed this function, I feel the need to point out that it says `lastest` which is absolutely not a word and should most likely say `latest`. `last` is also an option but I think whoever wrote it was going for `latest`